.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @arys-splunk
+pytest_splunk_addon/standard_lib/data_models @rkent-splunk @alexeisuv @justin-splunk
```

Comment:
@arys-splunk don't we have a group for that? it will be difficult to maintain

Reply:
Makes sense.